ml-notebooks/image_classification/pcam/02_1_vertex_ai_pcam.ipynb

```diff
@@ -0,0 +1,1604 @@
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "This notebook shows some examples of how to use [Vertex AI](https://cloud.google.com/vertex-ai/docs) for training a machine learning model. \n",
+    "\n",
+    "It shows how to define and submit a **model training job**; then how to upload and deploy that model for serving; and then how to send prediction requests to the deployed model's *Endpoint*. It also shows how to create and use a Managed Tensorboard instance during training, and how to log information about the training run to the Vertex Experiments API.\n",
+    "\n",
+    "Then, a follow-on notebook, `02_2_vertex_ai_pcam`, shows how to set up a [**hyperparameter tuning**](https://en.wikipedia.org/wiki/Hyperparameter_optimization) job using that same model; and how to set up and run a **distributed multi-node training** job.\n",
+    "\n",
+    "Then, a following set of notebooks show how to use Vertex Pipelines to define ML workflows for data preprocessing, training, model evaluation, and deployment.\n",
+    "\n",
+    "Currently, the code is on GitHub here: https://github.com/verily-src/terra-solutions-ml.\n",
+    "\n",
+    "### Estimated cost of running this notebook\n",
+    "\n",
+    "The dataset used for the examples in this notebook is fairly large, as is the base model architecture, and training using the notebook's default configurations will take about 1.5 hours.\n",
+    "\n",
+    "The model training works best with GPU(s)— it runs fine using only CPUs, but training will take an even longer time. For this example, the notebook itself doesn't need GPUs; instead they'll be used by Vertex AI.\n",
+    "\n",
+    "This example should cost < $2 to run, not including the cost of the notebook instance.\n",
+    "\n",
+    "### Running on a [Terra](http://app.terra.bio) notebook\n",
+    "\n",
+    "This example requires that TensorFlow >= 2.6 be installed, and does not require GPUs; instead the example uses GPUs on Vertex AI Training.\n",
+    "You can use the default GATK image. \n",
+    "\n",
+    "You will need to use a ['native' GCP project](https://support.terra.bio/hc/en-us/articles/360051229072-Accessing-advanced-GCP-features-in-Terra) to connect to the Vertex AI services.  The `00_pcam_setup.ipynb` notebook, which should be run before this one, will walk you through that setup.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "This notebook is not designed for running all the cells at once via a 'Run All'— rather, you will need to wait until training has finished to run the latter parts of the example.\n",
+    "    \n",
+    "If you like, you can shut down the notebook instance/Cloud Environment while the pipeline runs— monitoring its progress in the Cloud Console UI— and then restart the notebook instance when the job is finished to complete the example. If you do this, you'll need to do a bit of additional work to redefine some imports and config after the notebook is restarted.  \n",
```

Comment:
It'd be very nice to have a link (or a screenshot) to the relevant part of the Cloud Console UI.